common/utils/src/main/resources/error/error-classes.json

```
@@ -1983,6 +1988,11 @@
           "Expects group index between 0 and <groupCount>, but got <groupIndex>."
         ]
       },
+      "START" : {
+        "message" : [
+          "Expects `start` to start at 1 or start from the end if start is negative, but got <start>."
```

Actually, we expect non-zero argument, so, the parameter `<start>` is not needed. Let's rewrite this:
```suggestion
          "Expects a positive or a negative value for `start`, but got 0."
```